README.md

```diff
@@ -17,5 +23,70 @@ This mod requires MT 5.0.0 and above.
+`/smy` accept an invitation<br>
+
+## Settings
+
+All settings can be set in minetest.conf or accessed via mod with the global field of same name.
+
+- **spectator_mode.command_accept** (smy)<br>
+ After an invite has successfully been sent, the watcher may accept it with this command.
+- **spectator_mode.command_deny** (smn)<br>
+ After an invite has successfully been sent, the watcher may decline it with this command.
+- **spectator_mode.command_detach** (unwatch)<br>
+To stop observing another player, issue this command.
+- **spectator_mode.extra_observe_privs** ()<br>
+Additional privs granted to observers. e.g. noclip,never_hungry
+- **spectator_mode.extra_observe_privs_moderator** (<spectator_mode.extra_observe_privs>)<br>
+Additional privs granted to observers that used '/watch' command. e.g. jail,kick,teleport<br>
+If left empty, will use spectator_mode.extra_observe_privs
+- **spectator_mode.command_invite** (watchme)<br>
+To invite another player to observe player that issued this command.
+- **spectator_mode.command_attach** (watch)<br>
+To start observing another player, issue this command.
+- **spectator_mode.invitation_timeout** (60)<br>
+Invitations invalidate after this many seconds if they haven't been accepted or denied.
+- *spectator_mode.keep_all_observers_alive*** (false)<br>
+Not only moderators are kept oxygenated, but all observers when this is set to true.<br>
+For auto-feeding the spectator_mode.keep_alive() function needs to be overriden by another mod.
+- **spectator_mode.priv_invite** (interact)<br>
+The priv needed to send observation invites.
+- **spectator_mode.priv_watch** (watch)<br>
+The priv needed to silently observe any player that isn't currently watching another one.
+
+## Privelages
+
+Both privelages are registered if no other mod has already done so.
+
+## Compatibility
+
+Before sending invites, beerchat's player meta entry is checked to make sure muted players can't invite.<br>
+Other mods can override `spectator_mode.is_permited_to_invite(name_target, name_watcher)` to add own
+conditions of when who can invite whom.
+
+Moderators are kept breathing when observing via '/watch' command. Other mods can override this to
+add more functionality: `spectator_mode.keep_alive(name_watcher)`.
+
+`spectator_mode.on_respawnplayer(watcher)` can be overriden to adjust what happens when an attached player
+dies and respawns. Without change, the observer is detached for a split second then re-atached.
+
+While attaching a player, his hud flags are mostly turned off. Other mods can override the behaviour
+with their own implementation of<br>
+`function spectator_mode.turn_off_hud_hook(player, flags, new_hud_flags)`
+- **player** The PlayerObjectRef of player that is to be attached.
+- **flags** The player's HUD-flags prior to attaching.
+- **new_hud_flags** The table that can be manipulated and will be set as new flags.
+
+## Copyright
+
+Original mod DWTFYW Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
```

Comment:
Sam Hocebar is not my name.

Reply:
I'm sorry, I copied that straight from original licence file that had not changed since first commit 2821f9c ("add files", 2015-12-08)

I'll change that and email to the ones recorded with commit when I change to MIT.

Reply:
https://github.com/minetest-mods/spectator_mode/pull/15/commits/fe95d8cbf1a0c48ba3eed4f0a272e56fa55fc2d6